ddtrace/tracer/option_test.go

```diff
@@ -231,6 +231,48 @@ func TestLoadAgentFeatures(t *testing.T) {
 	})
 }
 
+func TestAgentIntegration(t *testing.T) {
+	t.Run("err", func(t *testing.T) {
+		assert.False(t, ImportIntegration("this-integration-does-not-exist"))
+	})
+
+	t.Run("default", func(t *testing.T) {
+		srv := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, _ *http.Request) {
+			w.Write([]byte(`{"endpoints":["/v0.6/stats"],"client_drop_p0s":true,"statsd_port":8999}`))
+		}))
+		defer srv.Close()
+		cfg := newConfig(WithAgentAddr(strings.TrimPrefix(srv.URL, "http://")))
+		assert.NotNil(t, cfg.integrations)
+		assert.NotEqual(t, len(cfg.integrations), 0)
```

Comment:
Does the `len` of integrations change? If not we can assert on the real value instead of just checking if it's not zero

Reply:
Shouldn't change unless we add more integrations! Makes sense to change it :)